### DISCUSSION TITLE
Change the prefix <leader>w to something else?

### DISCUSSION THREAD

**Discussion by catanzaromj** [lervag/wiki.vim#362] [Q&A] [answered]
Thanks for the awesome plugin.

Is it possible to change the prefix `<leader>w` to something else without having to copy/paste the full key mapping list? I was hoping to find something like a `wiki_prefix` option. Or is `<leader>w` hard-coded into the source file?

**Comment by lervag** [comment, marked as answer]
~Currently it is hard coded, sorry. But I guess I could change it to use a specified prefix.~

You can change it with `g:wiki_mappings_prefix`, see the docs in the latest update.

**Reply by lervag** [reply]
I.e. here: https://github.com/lervag/wiki.vim/commit/8d41b1e79aa0f0075729a253dcbd3c97583b3959

**Reply by catanzaromj** [reply]
Wow thank you for the quick reply and fix!!

**Reply by lervag** [reply]
No problem; I actually thought about this just recently. It was a quick fix and I only needed a nudge to actually act on it.
